### ISSUE TITLE
Update template for 1.0

### ISSUE THREAD

**Issue by Moini** [Neon22/inkscape_extension_template#7]
Hi @Neon22,

the extension template needs to be updated for Inkscape 1.0 .

More info is available here: 

https://wiki.inkscape.org/wiki/index.php/Release_notes/1.0#For_extension_writers

As this is universally useful to new extension authors, if you have a GitLab account, it could maybe be useful [to contact Martin / doctormo in the extensions repository bug tracker](https://gitlab.com/inkscape/extensions/issues), to see how to make it easier to find, and to have help with updating and including the new features (we can do images and links now, for example :) ).

**Comment by Neon22** [comment]
Yes I'm planning on getting back to this soon. Its summer down here now 
:)
Merry Xmas BTW.

I'll be developing in windoze alas and will have to find a suitable 
download.
Thanks, Mark...

**Comment by Moini** [comment]
Fortunately, finding an Inkscape program file for Windows isn't an issue anymore - they're auto-built via CI now. The latest ones are linked from https://inkscape.org/release/inkscape-1.0/?latest=1 (updated continuously, or at least once a day).

When I made this report, I forgot that I had an example inx file for some of the new features, maybe that'll help:  
https://gitlab.com/Moini/ink1.0talk/blob/master/demo_new_extension_layout.inx

The Python part, however, has changed a bit more ...  
Have a good Christmas time, too :)

**Comment by Moini** [comment]
@doctormo has now removed the link to this example repo from the website's extension development page. It's still a valuable info source, so please let us know when you update it, so we can check if it should be linked again to help new extension devs.

Hope all is well for you! :kiwi_fruit:
